lib/jwt_signed_request/verify.rb

```diff
@@ -29,7 +29,7 @@ def call
     attr_reader :request, :leeway
 
     def stored_key
-      jwt_header, _, _, _ = ::JWT.decoded_segments(jwt_token, false)
+      _body, jwt_header = ::JWT.decode(jwt_token, nil, false, algorithm: "None")
```

Comment:
Thanks for the PR, and yes using the `decode` method is a much better idea.

One small change is we can remove the `algorithm: "None"` from the argument list as we are not verifying it.

I tested it out and it returns the same thing 😄 

```
irb(main):003:0> jwt = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'

irb(main):004:0> JWT.decode(jwt, nil, false)
=> [{"sub"=>"1234567890", "name"=>"John Doe", "iat"=>1516239022}, {"alg"=>"HS256", "typ"=>"JWT"}]
```